astropy/units/format/cds.py

```diff
@@ -315,6 +308,12 @@ def _format_mantissa(cls, m: str) -> str:
     def _format_superscript(cls, number: str) -> str:
         return number if number.startswith("-") else "+" + number
 
+    @classmethod
```

Comment:
Ah, this is really a pity, that we now have to override the default a second time.

Reply:
This is not just changing the argument defaults, `CDS` should inherit this method from `Base` because `Generic` replaces it with the built-in `format()`.